specification/gp-connect-user-permissions.yaml

```diff
@@ -12,12 +12,14 @@ info:
     ## Overview
 
     Use this API to manage a patient's permissions to access the medical
-    record and services they can access at their GP practice.
+    record and services held at their GP practice.
 
     You can:
 
     - get a patient's permissions
+    - display only the sections that patients are permitted
     - request to update a patient's permissions to a higher level
+    - only request to update a patient's permissions to a level determined by the GP practice
```

Comment:
Similar to the comment above, I don't think this fits well here as it is a detail that should be included but should probably be included in either in the details for the update or close by.